src/adv.py

```diff
@@ -72,3 +165,15 @@
 # Print an error message if the movement isn't allowed.
 #
 # If the user enters "q", quit the game.
+
+# print starting room description to user
+# starting room is `Outside`
+print(f"\n\n {player.location['description']}")
+
+# grab the users input
+user_input = input(f"\n\nYou may either (Q)uit or head (N)orth into the {player.location['n_to']}: ")
+
+while (user_input.lower() != 'q'):
```

Comment:
i like the coercion to lowercase string, good for those that sometimes have capslock enabled!